runtime/src/block_cost_limits.rs

```diff
@@ -37,9 +37,9 @@ lazy_static! {
         (solana_sdk::stake::config::id(), COMPUTE_UNIT_TO_US_RATIO * 2),
         (solana_sdk::stake::program::id(), COMPUTE_UNIT_TO_US_RATIO * 25),
         (solana_config_program::id(), COMPUTE_UNIT_TO_US_RATIO * 15),
-        (solana_vote_program::id(), COMPUTE_UNIT_TO_US_RATIO * 85),
-        (secp256k1_program::id(), COMPUTE_UNIT_TO_US_RATIO * 4),
-        (system_program::id(), COMPUTE_UNIT_TO_US_RATIO * 10),
+        (solana_vote_program::id(), COMPUTE_UNIT_TO_US_RATIO * 70),
+        (secp256k1_program::id(), COMPUTE_UNIT_TO_US_RATIO * 2),
```

Comment:
what timing is this based on? Because the actual work of the secp256k1 program is not in the instruction execution part, it's executed in banking stage before it hands it to the message processor.

Reply:
The 2us is collected from per-program-timing metrics for secp256K1 (the dashboard linked in the issue #21917). So its actual number should come from banking stage?

Reply:
yea, the message processor part of it is like a no-op. We can cost it similar to a regular signature verify I think.

Reply:
Cool, changed it